erpnext/selling/sales_common.js

```diff
@@ -684,4 +684,33 @@ erpnext.selling.SellingController = class SellingController extends erpnext.Tran
 			}
 		})
 	}
+
+	add_update_vehicle_details_button() {
+		let me = this;
+		if (me.frm.doc.docstatus == 1 && me.frm.has_perm("submit")) {
+			me.frm.add_custom_button(__("Set Updated Vehicle Details"), function () {
```

Comment:
Relabel to Set Updated Applies to Details. Show button only if `applies_to_serial_no` is set